index.html

```diff
@@ -1,52 +1,27 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="utf-8">
-        <title>Anidata Starters</title>
+---
+layout: default
```

Comment:
See the comment in the anidata.github.io PR. Could we use a different layout for this? Would it be beneficial? Just a thought... I'm okay with either direction